src/store/chats/actions.js

```diff
@@ -0,0 +1,34 @@
+import { createAction } from 'redux-actions';
+import { newChat } from '../../api/chat';
+import * as ROUTES from '../../constants/routes';
+import { navigate } from '../navigation/actions';
+import { SET_CHAT, SET_CHATS } from './types';
+import { getMessages } from '../socket'
+
+
+const setChatAction = createAction(SET_CHAT);
+const setChatsAction = createAction(SET_CHATS);
+
+export function chatCreate(recieverId) {
+    return function (dispatch, getState) {
+        const userId = getState()
+        newChat(userId, recieverId)
+            .then(chat => {
+                dispatch(setChat(chat))
+            })
+    }
+}
+
+export function setChat(chat) {
```

Comment:
Always stick to the same style. Why are you using here function and at the top - the arrow function?
```
export const setChat = chat => dispatch => {
..
}
```